Source/Diagnostics/ReducedDiags/ParticleHistogram2D.cpp

```diff
@@ -60,6 +60,7 @@ ParticleHistogram2D::ParticleHistogram2D (std::string rd_name)
     ParmParse pp_rd_name(rd_name);
 
     pp_rd_name.query("openpmd_backend", m_openpmd_backend);
+    pp_rd_name.query("file_min_digits", m_file_min_digits);
```

Comment:
Awesome, please add to `parameters.rst` in the docs :)

Reply:
Ah, right, forgot that.

Thanks, added :)